2022-05-01 15:27:43.178 [Info] Discovered 0 plugins:


--------------------------------------------------------------------------------
2022-05-01 15:27:44.329 [Info] Connecting to 38.129.153.213:5400...

--------------------------------------------------------------------------------
2022-05-01 15:27:44.853 [Info] Connection approved [00000000-0000-0000-0000-000000000000].

--------------------------------------------------------------------------------
2022-05-01 15:27:44.939 [Debug] approval Time=7266131 Offset=63787008398697 Real=63787015664829

--------------------------------------------------------------------------------
2022-05-01 15:27:44.939 [Debug] local Time=7266131 Offset=63787008398807 Real=63787015664939

--------------------------------------------------------------------------------
2022-05-01 15:27:44.940 [Debug] real delta=109

--------------------------------------------------------------------------------
2022-05-01 15:27:44.940 [Debug] this.Statistics.Ping=0 NCPing=6

--------------------------------------------------------------------------------
2022-05-01 15:27:44.942 [Debug] Connection established to remote [00000000-0000-0000-0000-000000000000/38.129.153.213:5400].

--------------------------------------------------------------------------------
2022-05-01 15:27:44.942 [Info] Connected [00000000-0000-0000-0000-000000000000].

--------------------------------------------------------------------------------
2022-05-01 15:27:49.299 [Warn] 'Me' is null.

--------------------------------------------------------------------------------
2022-05-01 15:27:49.589 [Debug] received epp: 7270557

--------------------------------------------------------------------------------
2022-05-01 15:30:21.056 [Debug] received epp: 7422302

--------------------------------------------------------------------------------
2022-05-01 15:36:32.244 [Error] Received unhandled exception from System.Threading.Tasks.ContinuationTaskFromTask.

--------------------------------------------------------------------------------
2022-05-01 15:36:32.245 [Error] AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread.

Caused By
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-05-01 15:36:32.249 [Error] Caused by:

--------------------------------------------------------------------------------
2022-05-01 15:36:32.249 [Error] ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-05-01 15:36:32.250 [Info] Acquiring context lock... (0ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.250 [Info] Acquired. (0ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.251 [Info] Beginning context dispose. (0ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.252 [Info] GC.SuppressFinalize (0ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.252 [Info] InternalDispose() completed. (0ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.252 [Info] Clearing instance... (1ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.252 [Info] Releasing context lock... (1ms)

--------------------------------------------------------------------------------
2022-05-01 15:36:32.252 [Info] Released. (1ms)

--------------------------------------------------------------------------------
2022-05-01 15:37:54.871 [Error] InvalidOperationException: GetSkin: null
    Stack:    at Intersect.Client.Framework.Gwen.Control.Base.get_Skin()
   at Intersect.Client.Framework.Gwen.Control.Base.Dispose()
   at Intersect.Client.Interface.Debugging.DebugWindow.Dispose()
   at Intersect.Client.Interface.MutableInterface.DisposeDebugWindow()
   at Intersect.Client.Interface.Interface.DestroyGwen(Boolean exiting)
   at Intersect.Client.Core.Main.DestroyGame()
   at Intersect.Client.MonoGame.IntersectGame.Update(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.Tick()
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop()
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)
   at Intersect.Client.MonoGame.IntersectGame.MonoGameRunner.Start(IClientContext context, Action postStartupAction)
   at Intersect.Client.Core.ClientContext.InternalStart()
   at Intersect.Core.ApplicationContext`2.Start(Boolean lockUntilShutdown)


--------------------------------------------------------------------------------
